### ISSUE TITLE
Unarmed Strike not an available choice for Improved Natural Attack

### ISSUE THREAD

**Issue by zerkeros** [Heliomance/HeroForge-Anew#75]
Do the skill points calculate wrong or am I missing something? I build a monk with +0 INT modifier and the skill points per level show as 5 than 4. The book says that Monks get 4 + INT modifier per level, so I should have 4 not 5. Right?

**Comment by TwistedFaith84** [comment]
Are You A Human Monk?

**Comment by zerkeros** [comment]
Yes

**Comment by Zukibird** [comment]
Humans get +1 skill points/level, right?

**Comment by zerkeros** [comment]
Woops I totally forgot that O.o Stupid me.

**Comment by zerkeros** [comment]
Before locking/closing this issue, I have to report something. The "Improved Natural Attack" feat doesn't include "Unarmed Strike" in it. So as a Monk, I can't effectively choose it and see the effect at work.

Before disagreeing, here's the official response from Wizards on the matter: 
"This allows a monk access to all manner of weapon enhancements for her unarmed strikes. For example, a
monk can use the Improved Natural Attack feat to increase her unarmed strike damage."

**Comment by Heliomance** [comment]
Oh, bugger, I suspect that's likely to be a pain to fix. Natural attacks are calculated very differently from other weapons. Depending on where the INA dropdown pulls from, that's either going to be a really simple change or massively annoying.

**Comment by zerkeros** [comment]
So can we expect a bugfix soon, or this goes to v 8.0 "things to do"?

**Comment by Heliomance** [comment]
There's not going to be a bugfix release before v8.0. The current dev version is not in any fit state to actually use, and if I added it to the main release then there'd be a conflict when I tried to release v8.0. Spreadsheets don't diff properly, so you can't really merge changes from different branches.

**Comment by torrasque666** [comment]
It'll be real simple to add Unarmed Strike. It pulls from the Weapons stats page and it can be solved by just creating a new range one column to thr right of the old one (TblNaturalattacksCon i think) and telling the dropdowns to pull from that instead. I just did it last night, took me all of 5 minutes.

**Comment by Heliomance** [comment]
Yeah, except I've just changed how all that works, and there are no longer any calculations in the weapons stats page because it's been moved to a separate file.
